Sample Applications/ExpenseIt/ExpenseItDemo/Styles.xaml

```diff
@@ -166,6 +166,28 @@
         <Setter Property="MinHeight" Value="50" />
     </Style>
 
+    <Style x:Key="EmployeeListItemFocus" TargetType="Control">
+        <Setter Property="Control.Template">
+            <Setter.Value>
+                <ControlTemplate>
+                    <Rectangle StrokeThickness="1"
+                          Stroke="{DynamicResource {x:Static SystemColors.WindowTextBrushKey}}"
+                          StrokeDashArray="1 2"
+                          SnapsToDevicePixels="true"/>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Key="EmployeeListItem" TargetType="{x:Type ListBoxItem}">
+        <Setter Property="AutomationProperties.Name" Value="{Binding XPath=@Name}" />
+        <Style.Triggers>
```

Comment:
Seems to me this should work by default, so individual apps that use ListBox don't have to go through this trouble.  The same issue was raised in [bug 430176](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/430176), but the fix applied only to Aero theme (Win7), not to Aero2 or AeroLite (Win8, Win10).  I don't know why - perhaps it's one of those cases where Aero2 lacked the right extensibility points, but more likely just an oversight.

That fix set Margin="0,1,0,1" on the Rectangle, and used HighlightTextBrushKey instead of WindowTextBrushKey.  If those settings look OK in all the high-contrast themes, let's use them instead of yours.  For two reasons:  (1) consistency across OS, (2) if we change Aero2 theme defaults it will likely be to these values.

Please open a GitHub bug to consider fixing this for Aero2/Aerolite in .NET 6.0.

Reply:
Using HighlightTextBrushKey didn't work here. It makes the color of the stroke being the same of the background (Black), but I changed the margin of the Rectangle to match the one you said.